src/Microsoft.SqlTools.ServiceLayer/ShowPlan/Contracts/ExecutionPlanGraph.cs

```diff
@@ -23,7 +23,11 @@ public class ExecutionPlanGraph
         /// <summary>
         /// Underlying xml string used for generating execution plan graph
         /// </summary>
-        public string XmlString { get; set; }
+        public RawExecutionPlanGraph RawGraph { get; set; }
```

Comment:
1. The comment isn't correct anymore
2. Can this just be ExecutionPlanGraph? "Raw" isn't really accurate here since it's been parsed and whatnot already...